content/en/faq/terminal/typechain-guide.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Typechain-guide
+description: 'desc'
+draft: true 
+---
+Common functions or variables available on terminal :
+
+**Libraries**
+> 1) BigNumber
+> 2) axios
+> 3) DSA
+> 4) Web3
+> 5) wait
+
+**Constants**
+> 1) MAINNET_TOKENS
+> 2) POLYGON_TOKENS
+> 3) USER_ADDRESS
+> 4) ACCOUNT_ADDRESS
+
+**Library Instances**
+> 1) Web3
+> 2) DSA
+
+**You can also use top level await -**
+
+Usually you need async wrapper in JS:
+```js
+const run = async () => {
+   const gas = await web3.eth.getGasPrice();
+}
+
+run()
+ .then(() => console.log("Done"))
+ .catch((error) => console.log(error.message))
+```
+But we have top level await, no need to any wrapper
+```js
+const gas = await 
```

Comment:
plz can u put them in 1 line?

Reply:
put them what.. I didn't get this.. the white statement is a normal explanation .. of things and below is the code!

Reply:
```js
const gas = await 
web3.eth.getGasPrice();
```
=> 
```js
const gas = await web3.eth.getGasPrice();
```

Reply:
done